src/application-constants.ts

```diff
@@ -1,2 +1,3 @@
 export const SYNOPSYS_BRIDGE_DEFAULT_PATH_MAC = '/synopsys-bridge' //Path will be in home
+export const SYNOPSYS_BRIDGE_DEFAULT_PATH_WINDOWS = 'C:\\actions-runner'
```

Comment:
Not sure. If all user would have all the permission in the specified path. As we had discussed this could have been user's home path. 
And then synopsys-bridge to sync with other paths.